weasyprint/tests/test_draw/test_table.py

```diff
@@ -281,6 +281,45 @@ def test_tables_5():
     '''})
 
 
+@assert_no_logs
+@requires('cairo', (1, 12, 0))
+def test_tables_5_rtl():
+    assert_pixels('table_row_backgrounds', 28, 28, to_pix('''
+        ____________________________
+        _BBBBBBBBBBBBBBBBBBBBBBBBBB_
+        _B________________________B_
+        _B________________________B_
+        _B__bbbbbb_bbbbbb_bbbbbb__B_
+        _B__bbbbbb_bbbbbb_bbbbbb__B_
+        _B__bbbbbb_bbbbbb_bbbbbb__B_
+        _B__bbbbbb_bbbbbb_bbbbbb__B_
+        _B__bbbbbb_bbbbbb_bbbbbb__B_
+        _B__bbbbbb_bbbbbb_bbbbbb__B_
+        _B_________bbbbbb_________B_
+        _B__bbbbbb_ppppppbbbbbbb__B_
+        _B__bbbbbb_ppppppbbbbbbb__B_
+        _B__bbbbbb_ppppppbbbbbbb__B_
+        _B__bbbbbb_ppppppbbbbbbb__B_
+        _B__bbbbbb_ppppppbbbbbbb__B_
+        _B__bbbbbb_ppppppbbbbbbb__B_
+        _B________________________B_
+        _B_________rrrrrr_rrrrrr__B_
+        _B_________rrrrrr_rrrrrr__B_
+        _B_________rrrrrr_rrrrrr__B_
+        _B_________rrrrrr_rrrrrr__B_
+        _B_________rrrrrr_rrrrrr__B_
+        _B_________rrrrrr_rrrrrr__B_
+        _B________________________B_
+        _B________________________B_
+        _BBBBBBBBBBBBBBBBBBBBBBBBBB_
+        ____________________________
+    '''), tables_source % {'extra_css': '''
+      x-table { border-color: #00f; table-layout: fixed; direction: rtl }
```

Comment:
@liZe will we keep the direction at the table level or it should be on the body ?

Reply:
> @liZe will we keep the direction at the table level or it should be on the body ?

Putting `direction` on the table avoids a right-aligned table but keeps rtl columns and cells, that’s what we want to test.